securityconfig/allowlist.yml

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
The meta type of the 2 files are different. The `whitelist.yml` needs to be retained to support `whitelist` setting. It will be removed with `whitelist` setting in 3.0 though.